src/app/pages/login/login.component.scss

```diff
@@ -53,19 +53,28 @@ img.bg {
 }
 
 @media only screen and (max-width: 480px) {
+  .big-container{
+    display: flex;
+    width: 104%;
+    height: 32em;
+
+  }
   .container {
+    // display: flex;
```

Comment:
Hey @BrianSerem , nice work, kindly remove out this commented out code

Reply:
I have removed the pieces of commented code